app/helpers/observations_helper.rb

```diff
@@ -140,10 +140,10 @@ def name_html(naming)
       proposer_links = ""
     end
 
-    [name_link(naming), proposer_links].safe_join
+    [naming_name_link(naming), proposer_links].safe_join
```

Comment:
While I don't disagree with this renaming, what is it's relevance to this PR?

Reply:
My checklist erb was getting this `name_link` method when it was trying to call `name_link` from `object_link_helpers`. 

I suppose that helpers maybe can't have private methods?! 

So this was effectively a name collision, although in a different class. I'll do it in a separate PR